scalardb-test/src/main/java/scalardb/Common.java

```diff
@@ -1,4 +1,4 @@
-package verification.db.transfer;
+package scalardb;
```

Comment:
Not fully sure, but is it better to add `kelpie` or `kelpie-test` in the package ?

Reply:
Thank you! Sounds better.
I'll refine the package names for all tests in another PR.